contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/HdfsCopyStats.java

```diff
@@ -143,14 +153,18 @@ public HdfsCopyStats(String source,
                          boolean enableStatsFile,
                          int maxVersionsStatsFile,
                          boolean isFileCopy,
-                         HdfsPathInfo pathInfo) {
+                         HdfsPathInfo pathInfo,
+                         String storeName) {
         this.sourceFile = source;
         this.totalBytesTransferred = 0L;
         this.bytesTransferredSinceLastReport = 0L;
         this.pathInfo = pathInfo;
+        this.storeName = storeName;
         this.lastReportNs = System.nanoTime();
         this.startTimeMS = System.currentTimeMillis();
         initializeStatsFile(destination, enableStatsFile, maxVersionsStatsFile, isFileCopy);
+        // Register the current HdfsCopyStats to the aggregated stats
+        HdfsFetcherAggStats.getStats().addStoreCopyStats(storeName, this);
```

Comment:
instead of calling , getStats method on each one of the invocation, I would prefer to have the Static variable here.